.github/workflows/build.yml

```diff
@@ -44,10 +45,13 @@ jobs:
       run: nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
     - name: Setup cachix
       uses: cachix/cachix-action@v10
+      # Don't replace <YOUR_CACHIX_NAME> here!
       if: ${{ matrix.cachixName != '<YOUR_CACHIX_NAME>' }}
       with:
         name: ${{ matrix.cachixName }}
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+        # or:
+        #authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
```

Comment:
Is there a way to enable this conditionally on whether the secret exists or not?

Reply:
I don't know of a way. If I read [the reference](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#context-availability) right the secrets context isn't available in `jobs.<id>.steps.if` and I don't know other forms of conditionals.

Reply:
Hmm, I wonder what happens if we include both. If cachix/cachix-action/github is smart enough to not use an empty value it'd be perfect

Reply:
Seems to work just fine, [tested it here](https://github.com/toonn/nur-packages/runs/3114040358?check_suite_focus=true) in a repo without `CACHIX_SIGNING_KEY` set.